api-docs/client-interactions.md

```diff
@@ -0,0 +1,198 @@
+# Client Interactions
+
+The client interactions document direct encounters with the clients.
+
+Here is the list of the valid interaction types:
+
+- `clientInteractionCreated`,
+- `clientInteractionUpdated`,
+- `clientInteractionDeleted`,
+
+## Get a single Client Interaction
+
+### Request
+
+```http
+GET /api/clients/:id/interactions/:id
+```
+
+### Response
+
+```json
+{
+  "id": 1,
+  "title": "Nutrition Program - SNAP",
+  "interactionType": "walk-in",
+  "notes": "Application successfully filled out. Decision pending.",
+  "startTime": "2019-05-06T05:00:00.000Z",
+  "endTime": "2019-05-06T06:00:00.000Z",
+  "location": "CU Office",
+  "referenceServices": ["Citizenship", "Financial Coach"],
```

Comment:
Remove this -- we're not doing referrals for Phase 2

Reply:
Change was added on this last night.